addons/medical_treatment/initSettings.sqf

```diff
@@ -25,6 +25,15 @@
     true
 ] call CBA_settings_fnc_init;
 
+[
+    QGVAR(woundReopenChance),
+    "SLIDER",
+    [LSTRING(WoundReopenChance_DisplayName), LSTRING(WoundReopenChance_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [0, 2, 1, 2],
```

Comment:
Maybe increase the upper limit to 5.

Reply:
I feel bad for the unit that wants it that high! Done. Thanks!